### DISCUSSION TITLE
Relationship in a table that refers to itself

### DISCUSSION THREAD

**Discussion by DZ8540** [adonisjs/core#3895] [Help]
Hi there, i have posts comments table, and this table has a postComment_id column, which has relation to itself.

<img width="190" alt="image" src="https://user-images.githubusercontent.com/50152205/188172358-5808e39c-6a95-46f4-9f38-c8f8901d90d5.png">

And when this column is not empty its mean what this row is reply and not comment.
So, if row has column which foreign to reply (postComment_id = reply_id) it's mean reply to reply.

In my model i create relations below.

<img width="601" alt="image" src="https://user-images.githubusercontent.com/50152205/188172844-30e5e5aa-f1e8-4b3a-afbe-c6377994af23.png">

And when i use preload reply relation, it's preloaded only replies, and not replies to replies.

<img width="814" alt="image" src="https://user-images.githubusercontent.com/50152205/188173893-e5cfd490-2ab8-4a3a-990e-55677f31a091.png">

It's returned toSQL() method:

<img width="497" alt="image" src="https://user-images.githubusercontent.com/50152205/188175601-42add52e-7f12-462e-bebe-8eb15633acab.png">

What i doing wrong?

**Comment by thetutlage** [comment]
If I get the question right, you are not able to load the nested replies? Or not even the top level replies?

**Reply by DZ8540** [reply]
Yep, exactly!

**Comment by SkifAlef** [comment]
If it still actual, rename `postComment_id `column in DB to `post_comment_id`

**Reply by DZ8540** [reply]
OK, i will try this later, thanks
